source/exe/stripped_main_base.h

```diff
@@ -36,6 +36,8 @@ class ProdComponentFactory : public Server::ComponentFactory {
 // separate for legacy reasons.
 class StrippedMainBase {
 public:
+  virtual ~StrippedMainBase() {}
```

Comment:
nit: `virtual ~StrippedMainBase() = default;`
Clang-tidy should complain this, maybe I haven't turn it on now.